mininterface/tk_interface/utils.py

```diff
@@ -67,12 +68,89 @@ def get(self):
 
 
 def choose_file_handler(variable: Variable, tag: PathTag):
+    """Handler for file/directory selection on PathTag"""
     def _(*_):
-        if tag.multiple:
-            out = str(list(askopenfilenames(title="Choose files")))
+        initialdir = os.getcwd()
+
+        # Check whether this is a directory selection
+        if tag.is_dir:
+            # Directory selection using askdirectory
+            kwargs = {"title": "Select Directory", "initialdir": initialdir}
```

Comment:
It seems to me the tkinter already uses the default directory cwd. If you'd like to change the init directory (which is a great improvement), I'd use the path in the tag.val.